applications/Preprocessor/lib/MeshSource.h

```diff
@@ -159,8 +163,17 @@ class MeshSource2 {
          */
         std::string zoneName;
 
+        /**
+         * Dimension of the Element
+         */
+        size_t dimension;
+        /**
+         * Index of the Element
+         */
+        size_t id;
```

Comment:
I think it may be good to clarify that this is just as courtesy for implementing a reader. That this id is not required, and should not be used in any further processing steps.

Reply:
Yes